### ISSUE TITLE
Documentation: Describe the packaging process for backdrop-contrib

### ISSUE THREAD

**Issue by indigoxela** [backdrop/backdrop-issues#4141]
Someone on backdrop-contrib had a problem with a release. Unfortunately I didn't find the appropriate documentation for the packaging process.

If it exists, it should get a link somewhere here: https://api.backdropcms.org/modules
Currently the docs say:

> version (Optional)
> version = 1.0.2
> 
> You can give your module whatever version string is appropriate.

That's somewhat misleading regarding releases on backdrop-contrib.

If docs for automatic packaging don't exist yet - shouldn't we add it somewhere?

For the records - this is the issue: https://github.com/backdrop-contrib/b/issues/66

**Update:** there is some documentation https://api.backdropcms.org/documentation/contribute-module-layout-or-theme
But it doesn't mention that the release version gets added automatically and shouldn't get set in the .info file.

**Comment by AlexShapka** [comment]
Yes, please document this in somehow noticeable way as otherwise newcomers like me will be hitting the same issue described on backdrop-contrib/b#66
